proto/meta.proto

```diff
@@ -55,6 +55,7 @@ message TableFragments {
   map<uint32, Fragment> fragments = 2;
   map<uint32, ActorStatus> actor_status = 3;
   repeated uint32 internal_table_ids = 4;
+  map<uint32, uint32> parallelisms = 5;
```

Comment:
This information looks duplicated with actors? You can get parallelism by looking at how many actors this fragment has.